src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public enum Color {
+    BLACK("\u001B[30m"),
+    RED("\u001B[31m"),
+    GREEN("\u001B[32m"),
+    YELLOW("\u001B[33m"),
+    BLUE("\u001B[34m"),
+    PURPLE("\u001B[35m"),
+    CYAN("\u001B[36m"),
+    WHITE("\u001B[37m");
+
+    private static Random random = new Random();
+    private String code;
+
+    Color(String code) {
+        this.code = code;
+    }
+
+    public static Color getRandom() {
+        switch (random.nextInt(8)) {
```

Comment:
8 is a magic number, why not 7 or 9 ?

Reply:
We have 8 different color in enum structure, so i used random.nextInt(8) which return [0; 8). It is better to change it to variable? If do that, may appeaar situation, when someone add the new color but forget add it to random generator

Reply:
so instead you can use enum length which will make code more readable and flexible 
https://stackoverflow.com/questions/17492079/how-to-get-number-of-possible-item-of-an-enum/17492102